src/containerapp/azext_containerapp/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore =
+     W503   # line break before binary operator, not compliant with PEP 8
+     E203   # whitespace before ':', not compliant with PEP 8
```

Comment:
Why don't you fix the issue of the code style directly but ignore the flake8 checks?

Reply:
@haroonf can you remove these changes as well - will re-visit this once Silas is back.